client/src/components/basic/task-instructions/components/Rank/rank.module.scss

```diff
@@ -0,0 +1,28 @@
+:local(.rank) {
```

Comment:
file or folder name should follow [JS quality criteria](https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/source/javascript.md#naming)